### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by ili**
# Features
- Full set of joins supported (#685, #854)
- `Query.ClearCache()` method (#898)
- SQLite FTS support (#893, #773)
- New Oracle BulkCopy method (#878)
- `Sql.NoConvert` method (#870, #722)
- Expression predicate for associations (#841)
- Expression post processing (#852)

# Fixies
- Extension associations (#865, #833 )
- Firebird Merge (#851, #881)
- Oracle DateTime2 convert (#879)
- Database schema (#871, #858 )
- PostgreSQL datepart (#882, #883)
- MS Sql stored procedure XML parameters (#859)
- Query caching (#862, #863, #903)
- Group By (#264)
- SerializableAttribute removed from linq2db.core (#839)
- Inheritance mapping (#895, #899)
- Configuration (#906)
- NULL check for `IN` in subquery (#909)

# Significant
- Signing removed for linq2db.core (#867)
- Query now executed before result is returned (previously query was executed on IEnumerable iteration) (#884, #872)
- IDataContextEx removed (#837, #849)

# Improvements
- Join optimization (#834)
- Documentation (#836)
- Async extensions (#838)

# Links
- [Milestone 1.10.0](https://github.com/linq2db/linq2db/milestone/6?closed=1)

@linq2db/testers please run tests

### CODE REVIEW

**Comment by detoxhby** [comment]
Nice features!

As always, I'm asking for the time window available for testing. Thanks!

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Partial code listings, or multiple fragments of code, will slow down our respons
 ```
 
 ### Environment details
-linq2db version: *1.8.3*  
+linq2db version: *1.9.0*
```
Comment (sdanyliv) [inline .github/ISSUE_TEMPLATE.md]: Correct version

**Comment by RoyC-Retail-Assist** [comment]
I've run my regression tests on the iSeries DB2 provider and everything passes.

Great work.  Thank you.

**Comment by detoxhby** [comment]
✅ Test suite on SQL CE 4.0 and runtime on SQL Server 2014-2017 works fine. 👍

**Pull request merged**